docs/gettingstarted/quickstart.rst

```diff
@@ -0,0 +1,313 @@
+:layout: homepage
+
+============================
+Getting started with Flocker
+============================
+
+.. logo::
+
+.. intro-text::
+
+   Flocker lets you run microservices apps with database containers and move them around between servers. It comes in two pieces and you’ll need both.
+
+.. contents::
+   :local:
+
+.. tutorial-step::
+
+   Step 1: Installing Flocker CLI & Node
+   =====================================
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Local
+
+      .. image:: images/macbook.png
+         :class: center-block img-responsive
+         :alt: Flocker CLI diagram
+
+      Flocker CLI
+      -----------
+
+      +--------------------------------------------------------------------------+
+      | Runs on your laptop                                                      |
+      +--------------------------------------------------------------------------+
+      | Uses application and deployment configuration files                      |
+      +--------------------------------------------------------------------------+
+      | Deploys containers to a cluster of servers                               |
+      +--------------------------------------------------------------------------+
+
+      .. empty-div:: arrow-down center-block hidden-xs hidden-sm
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Live
+
+      .. image:: images/nodes.png
+         :class: center-block offset-top img-responsive
+         :alt: Flocker Node diagram
+
+      Flocker Node
+      ------------
+
+      +--------------------------------------------------------------------------+
+      | Runs on each server in a cluster                                         |
+      +--------------------------------------------------------------------------+
+      | Links, ports and volumes work across hosts                               |
+      +--------------------------------------------------------------------------+
+      | After deployment, containers can move around                             |
+      +--------------------------------------------------------------------------+
+
+      .. empty-div:: arrow-down center-block
+
+   .. parallel::
+
+      Installation
+      ------------
+
+      .. mobile-label::
+
+         Local
+
+      .. noscript-content::
+      
+         OS X
+         ^^^^
+
+         Install the flocker-cli client on your Mac (requires Homebrew):
+
+         .. code-block:: console
+
+            $ brew update && \
+              brew tap clusterhq/flocker && \
+              brew install flocker-0.3.2
+            
+      .. noscript-content::
+
+         Ubuntu / Debian
+         ^^^^^^^^^^^^^^^
+
+         Install the dependencies:
+         
+         .. code-block:: console
+         
+            $ sudo apt-get install gcc python2.7 python-virtualenv python2.7-dev
```

Comment:
Problems I encountered:

  * my user is not a sudoer so this step failed (fixed by getting root some other way)
  * my package list is out of date so apt-get couldn't find the packages (fixed by running `apt-get update`)
  * `apt-get install` prompted me to continue (fixed by running `apt-get install -y ...`)

I think the first is "too bad" there's nothing we can reasonably do about that.
I think the second two points can reasonably be addressed by changing the command we suggest.

Also I think the intent of these panels is to provide a single command the user can copy/paste to get Flocker so please combine the fixed version of this command with the rest of the install steps with `&&` as is used below.